docs/circuit-test.md

```diff
@@ -44,6 +44,49 @@ View a history of completed circuit tests, including the provider the call route
+|**_100_183_Delay**|Delay between 100 Trying and 183 Session Progress|
+|**Audio Signal Delay (Latency)**| Time it takes for an audio signal to travel from the sender to the receiver. Its a crucial factor in the quality of voice and video communication in telecommunication systems|
+|**Latest RBT Delay**|The delay between selecting a Ring Back Tone (RBT) and it actually playing for callers can vary significantly|
+|**Early Media Peak Signal Level**|The highest volume level measured during the initial audio exchange before the call is fully connected. This information could be useful for QoS, Network troubleshooting, Media optimization|
+|**Active Media Peak Signal Level**|Refers to the highest power level attained by the audio or video signal during a call or streaming session| This information could be useful for Audio/video quality, Network performance, Regulation compliance|
+|**Recorded**| Recorded received call file name, transmitted call file name, mix file name|
+|**Answered Duration**| Duration of the answered call|
+|**Current Calls Count**|Number of current calls|
+|**SIP Pcap FileName**|Filename of a SIP Pcap (packet capture) file|
+|**SIP and RTP Pcap FileName**|Filename of a SIP and RTP Pcap (packet capture) file|
+|**RTP Codec**|Name of the RTP Codec used in the call|
+|**RTP Caller**|1. **Address** of the caller. <br> 2. **Port** used by the caller for call sessions. <br> 3. **Synchronization Source (SSRC)** identifier for a specific source of synchronized multimedia data within an RTP session <br> 4. **G107R** is a codec <br> 5. **G107MOS** is a codec <br> 6. **Packets Count**: Number of packets. <br> 7. **Lost Packets**: Number of packets lost in transmission <br> 8. **Max RFC3550 Jitter**: Refers to the maximum value of the calculated jitter for an RTP stream over a specified measurement interval. High jitter can negatively impact the quality of real-time applications such as voice and video calls, as it can lead to stuttering, delays, and distortions. <br> 9. **Mean RFC3550 Jitter**: Refers to the average deviation of the time it takes for consecutive RTP packets to arrive at the receiver, compared to the sender's timestamps. It's a measure of the variability in the packet arrival times, which can be caused by various factors such as network congestion, routing changes, and packet loss. <br> 10. **Max Delta**: refers to the largest difference in arrival times between two consecutive RTP packets received at the receiver. It essentially measures the maximum variation in the time it takes for packets to arrive, which can be an indicator of network issues like congestion or variable latency. <br> 11. **Min Delta**: Minimum time difference between RTP packets. <br> 12. **Packet Time**: The time packet was sent or received <br> 13. **Non Silent Percentage**: Proportion of time during a call when there is actual conversation happening <br> 14. **Delay**: Time it takes for an RTP packet to travel from the sender to the receiver|
```

Comment:
Max Delta: refers    Refers   ( R is capital)

Reply:
done